config/kairosdb.properties

```diff
@@ -54,6 +54,7 @@ kairosdb.reporter.ttl=0
 
 # Configuring tagging of self-instrumentation metrics
 kairosdb.reporter.tagger.rest.query.class=org.kairosdb.core.reporting.NoTagsTagger
+kairosdb.reporter.tagger.rest.datapoints.class=org.kairosdb.core.reporting.NoTagsTagger
```

Comment:
I think we will also need to modify the config file in our repo? (configs/observability/amp/kairosdb/config/kairosdb.properties)

Reply:
Absolutely. That will be done separately after this is released and imported.